### ISSUE TITLE
Suggestion - Need treatment notification

### ISSUE THREAD

**Issue by Dunkhan** [skyarkhangel/Hardcore-SK#700]
The notification that a colonist need medical treatment is not shown if the injured/sick colonist is in a hospital bed. I do not like this feature at all. I would like to be notified immediately when treatment is required regardless of the bed a colonist is in.

Note: I know that suggestions are not always useful when people have other priorities, including fixing game breaking bugs. I have submitted this partly to check interest and may find time to implement it myself if it is wanted. Please post here if you think this is a sensible change and/or wish to implement it.

Edit: I was bored after posting this so I looked into the code. Neither ILSpy nor Telerik JustDecompile can correctly decompile the relevant section of code so I will not be able to implement this. As a result I have reported this behaviour as a bug on the official forum: https://ludeon.com/forums/index.php?topic=30217.0

Feel free to close this report, otherwise I will do it myself once I am sure there is no chance of a mod solving the issue.

**Comment by skyarkhangel** [comment]
Yep seems its vanilla bug, i hope fixed in a17

**Comment by Dunkhan** [comment]
Thanks for the reply. I tried making a custom alert just for myself because I don't want to wait till a17, but it didn't work. Details here https://ludeon.com/forums/index.php?topic=30440
I don't know if I did something wrong or if the bug that causes the issue also affected my code.

Either way the new status icons in this mod go a long way as there is a 'needs treatment' status icon, so that is something at least.

I will close this, as it is not a problem that this community has to deal with. Thanks for all your hard work, loving the newest changes.
